apex/parallel/distributed.py

```diff
@@ -104,7 +104,7 @@ class Reducer(object):
     When used with this launcher, :class:`Reducer` assumes 1:1 mapping of processes to GPUs.
     It also assumes that your script calls ``torch.cuda.set_device(args.rank)`` before creating the model.
 
-    main_reducer.py in https://github.com/NVIDIA/apex/tree/master/examples/imagenet shows example usage.
+    main_amp.py in https://github.com/NVIDIA/apex/tree/master/examples/imagenet shows example usage.
```

Comment:
main_amp.py only shows DDP usage.  We don't have an example for Reducer, it's more of a tool for internal edge cases, and we don't intend external people to use it.